.pre-commit-config.yaml

```diff
@@ -20,7 +20,8 @@ repos:
     hooks:
       - id: hadolint-docker
         # ignore "using latest" warning
-        entry: hadolint/hadolint:v2.10.0 hadolint --ignore DL3007
+        # ignore "package==<version>" warning
+        entry: hadolint/hadolint:v2.10.0 hadolint --ignore DL3007 --ignore DL3008 -t warning
```

Comment:
You have it in the hook, which is good. Even better would be to also check in the CI pipeline, just in case someone does not install the hooks